src/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -3,9 +3,31 @@
 
```

Comment:
I think the POR is to not have/use components CSS, you should port everything over instead and make it RTL aware